### DISCUSSION TITLE
Pull data from CrunchBase API

### DISCUSSION THREAD

**Discussion by hugoroy** [tosdr/tosdr.org#138] [Ideas]
We should be able to link our data to CrunchBase API

http://www.crunchbase.com/organization/duck-duck-go

should be linked from https://tosdr.org/services/duckduckgo.json

A lot of useful features can work if we do this. For instance, we could pull the description given by crunchbase: "    DuckDuckGo is a search engine that supports private browsing and offers personalized search results for its users." and put it in the json file -- so that users searching on tosdr.org could find things based on this data.

**Comment by mrshu** [comment]
Quick note: It would be necessary to get access to their API by signing up for an account [here](https://developer.crunchbase.com/)

**Comment by hugoroy** [comment]
Shit, they changed their license:
http://info.crunchbase.com/docs/terms-of-service/

 Certain parts of the CrunchBase Dataset may contain contributions made prior to December 12, 2013, and those contributions will continue to be governed by the prior CrunchBase Terms of Service referencing the Creative Commons Attribution License [CC-BY].

We need to look for a fork of the data under a license we can use.

We cannot use the non-commercial licenses.
